docs/src/performance_tips.md

```diff
@@ -138,5 +135,16 @@ include(t)
 ```
 
 We see that Unions of up to three different Agent types do not suffer much.
-Hence, if you have less than four agent types in your model, using different types is still a valid option and allows you to utilize multiple dispatch.
-For more agent types however we recommend using the [`@multiagent`](@ref) macro.
+Hence, if you have less than four agent types in your model, using different types is still a valid option.
+For more agent types however we recommend using the [`@sumtype`](@ref) macro.
+
+Finally, we also have a more realistic benchmark of the two approaches at `test/performance/sum_faster_than_multi.jl` where the
```

Comment:
What makes this benchmark more realistic?

Reply:
Types are more heterogeneous as I would expect in a multi agent simulation, also it is more complex to optimize away dynamic dispatch in this setting